CONTRIBUTING.md

```diff
@@ -7,88 +7,24 @@ CHAOSS Slack Bot is an open source project and we love to receive contributions
 
 We follow the standard Git workflow of fork -> change -> pull request -> merge -> update fork -> change ... (repeat forever). If you are new to open source, we recommend GitHub's excellent guide on "How to Contribute to Open Source". In addition, please feel free to reach out to any of the maintainers or other community members if you are struggling; we are here to help you learn!
 
-# Your First Contribution
-**Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://kcd.im/pull-request).
-At this point, you're ready to make your changes! Feel free to ask for help; everyone is a beginner at first. 😸
+## Adding or editing wiki pages locally
```

Comment:
So we've tested this workflow, and it only works for admins. Please use this instead:

## Contributing to the wiki
- Fork the repo
- Navigate to https://github.com/<YOUR_USERNAME>/chaoss-slack-bot/wiki
- Hit ```Create the first page``` (You may get an error, that's okay)
- Navigate to https://github.com/<YOUR_USERNAME>/chaoss-slack-bot/wiki again and you will see the wiki updated.
- Clone the forked repo 
```
git clone https://github.com/<YOUR_USERNAME>/chaoss-slack-bot/wiki.git
```
- Add the upstream:
```
git remote add upstream https://github.com/chaoss/chaoss-slack-bot.wiki.git
```
- Fetch the latest updates from the staging branch:
```
git fetch upstream staging
```

**Keeping your fork in sync**
```
git checkout staging
git fetch upstream staging
git merge upstream/staging
git push origin staging
```

**Submitting changes to your fork**
```
git checkout staging
git add .
git commit -s -m "descriptive commit message"
git push origin staging
```
**Submitting changes to the main repository**
When you're ready to submit, you need to [open an issue](https://github.com/chaoss/chaoss-slack-bot/issues) and provide the following:
- Link to your repository.
- Description to the change you've done.

N/B: If you're fixing an existing issue, you don't need to open a new one. Just submit your changes there.

Reply:
Hi @ElisaCapololo, please update your branch by pulling the latest changes from `staging`